kafka-base/Dockerfile

```diff
@@ -11,21 +11,32 @@ RUN groupadd -r -g 1001 kafka && useradd -r -m -u 1001 -g kafka kafka
 # Set Scala and Kafka version
 ENV SCALA_VERSION=2.11
 ENV KAFKA_VERSION=1.0.0
+ENV JMX_EXPORTER_VERSION=0.1.0
 
-# Set Kafka SHA512 checksum
-ENV CHECKSUM="b698028485d460c494af942b2dfa41c7342a2ad6052b543c95289cd1e832a1ebcf679b7e568dcabc342c799765337b94f91bc0cf0ee915534cdf82d1635a622a  kafka_2.11-1.0.0.tgz"
+# Set Kafka (SHA512) and Prometheus JMX exporter (SHA1) checksums
+ENV KAFKA_CHECKSUM="b698028485d460c494af942b2dfa41c7342a2ad6052b543c95289cd1e832a1ebcf679b7e568dcabc342c799765337b94f91bc0cf0ee915534cdf82d1635a622a  kafka_2.11-1.0.0.tgz"
+ENV JMX_EXPORTER_CHECKSUM="6ab370edccc2eeb3985f4c95769c26c090d0e052 jmx_prometheus_javaagent-0.1.0.jar"
```

Comment:
We could use the `JMX_EXPORTER_VERSION` variable in this line too.

Reply:
@tombentley I was thinking exactly the same thing even about the KAFKA_CHECKSUM env var for SCALA_VERSION and KAFKA_VERSION so I decided to use the same way as @scholzj. But ... in any case the first part (the checksum actually) is strictly related to the version, so in any case the env var should be changed. Btw I agree that it could be worth using these env vars for building the checksum one. Wdyt ?